features/ui/form/select/select.tsx

```diff
@@ -0,0 +1,189 @@
+import React, {
```

Comment:
Wow, great job building this from scratch. Any particular reason why you didn't use something like `react-select`? Seems to work great though in Storybook 👍

Reply:
Thanks. No particular reason - I wasn't aware of `react-select`